environment.yml

```diff
@@ -15,7 +15,7 @@ dependencies:
   - matplotlib=3.5
   - notebook
   - iminuit>=2
-  - joblib
+  - joblib==1.2
```

Comment:
I think this will pin to `1.2.0`, we should pin to `1.2.x` I think. `joblib=1.2` here

Reply:
does it? 🤔 
Damn, ok changing